00-linux-basics/Exercises.md

```diff
@@ -0,0 +1,51 @@
+# Ejercicios
```

Comment:
Please, ignore Exercises.md file because it only contains the exercises without the solutions